src/graphql/mutations/Chat.ts

```diff
@@ -18,6 +18,7 @@ export const CREATE_AND_SEND_MESSAGE_MUTATION = gql`
           label
         }
         media {
+          id
```

Comment:
You've added Media Id, but except for mock data, it is not used anywhere.

Reply:
@DigneshGujarathi I was earlier using the id for updating the image but later used ref. Removed all media Id's.